src/test/java/com/wikia/webdriver/elements/fandom/components/LoadMore.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.elements.fandom.components;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import org.openqa.jetty.html.List;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.jetty.html.List'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)